Client/Frontend/Browser/SearchEngines/SearchEngines.swift

```diff
@@ -40,6 +41,7 @@ class SearchEngines {
     private var engineProvider: SearchEngineProvider
 
     weak var delegate: SearchEngineDelegate?
+    private var logger: Logger = DefaultLogger.shared
 
     init(prefs: Prefs, files: FileAccessor, engineProvider: SearchEngineProvider = DefaultSearchEngineProvider()) {
         self.prefs = prefs
```

Comment:
I'd prefer if logger was part of the init, similar to how `BundleImageFetcher.swift` has it, for next time.